openapi.go

```diff
@@ -23,18 +26,21 @@ func newOpenAPI(file string) *openAPI {
 	return &openAPI{
 		swagger,
 		openapi3filter.NewRouter().WithSwagger(swagger),
+		logger,
 	}
 }
 
+// getRoute gets the openapi3filter.Route instance related to the given URL
 func (o *openAPI) getRoute(url *url.URL) *openapi3filter.Route {
 	route, _, err := o.router.FindRoute("GET", url)
 	if err != nil {
-		log.WithFields(log.Fields{"url": url, "reason": err}).Debug("Route not found in the OpenAPI specification")
+		o.logger.Debug("route not found in the OpenAPI specification", zap.Stringer("url", url), zap.Error(err))
```

Comment:
```suggestion
		o.logger.Debug("Route not found in the OpenAPI specification", zap.Stringer("url", url), zap.Error(err))
```

Reply:
it's better to use lower case everywhere to be consistent with what's done by Caddy.